source/daplink/cmsis-dap/DAP_vendor.c

```
@@ -382,7 +382,14 @@ uint32_t DAP_ProcessVendorCommand(const uint8_t *request, uint8_t *response) {
 #endif /* INTERFACE_STM32H743 */
         break;
     }
-    case ID_DAP_Vendor20: break;
+    case ID_DAP_Vendor20:
+    {
+        // Request to stay in bootloader mode on the next boot for SWU
+        *reponse = DAP_OK;
+        config_ram_set_hold_in_bl(true);
```

how to trigger a reboot via a command?

---

I was under the impression the host device can power cycle the USB? If that's not the case, I can make this a "reset and hold in bl" command.

---

Many setups will have a "cambrionix" usb hub with power cycling ability, but I think developer setups are less likely to have that.  
A "reset and hold in bl" will be simpler to use I think.  I guess the usb connection to the daplink will go down, but I think the ndm code can ignore any exceptions, open a new connection, etc.